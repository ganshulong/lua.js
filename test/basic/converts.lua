print("1"+2)
print(pcall(function()
    print("a"+2)
end))
