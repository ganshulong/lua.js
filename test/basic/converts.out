3
false Error: attempt to perform arithmetic on a string value
